build.gradle

```diff
@@ -227,38 +228,28 @@ android {
         includeInBundle = false
     }
 
-    // The flavor dimensions for build variants (e.g. aospWithQuickstep, aospWithoutQuickstep)
     // See: https://developer.android.com/studio/build/build-variants#flavor-dimensions
-    flavorDimensions += ["app", "recents"]
+    flavorDimensions += ["app", "recents", "channel"]
 
     productFlavors {
-        aosp {
-            dimension "app"
-            applicationId 'com.android.launcher3'
-            testApplicationId 'com.android.launcher3.tests'
-        }
-
-        l3go {
-            dimension "app"
-            applicationId 'com.android.launcher3'
-            testApplicationId 'com.android.launcher3.tests'
-        }
-
         lawn {
             dimension "app"
-            applicationId 'app.lawnchair'
-            testApplicationId 'com.android.launcher3.tests'
-            isDefault true
         }
 
         withQuickstep {
             dimension "recents"
             minSdk 26
-            isDefault true
         }
 
-        withoutQuickstep {
```

Comment:
@Goooler @suphon-t don't we need this for normal non-quickstep users eg playerstore?